examples/rust/rideshare/server/src/main.rs

```diff
@@ -49,21 +47,41 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         vars
     });
 
+    let (add_tag, remove_tag) = agent_running.tag_wrapper();
+    let add = Arc::new(add_tag);
+    let remove = Arc::new(remove_tag);
+
     // Bike Route
-    let bike = warp::path("bike").map(|| {
+    let bike = warp::path("bike").map(move || {
+        add("vehicule".to_string(), "bike".to_string());
```

Comment:
can you change all instances of `vehicule` to the proper spelling of `vehicle` (unless there is a reason you're spelling it differently)

Reply:
Good catch! (it's the French spelling)